pkg/metrics/metrics.jsx

```diff
@@ -1824,14 +1842,11 @@ class MetricsHistory extends React.Component {
 
         // on a single machine, cockpit-pcp depends on pcp; but this may not be the case in the beiboot scenario,
         // so additionally check if pcp is available on the logged in target machine
-        if ((cockpit.manifests && !cockpit.manifests.pcp) || this.pmlogger_service.exists === false)
+        if (!this.state.isPythonPCPInstalled || this.pmlogger_service.exists === false)
```

Comment:
See above, please compare to `=== false` to not show this message while initializing the page.